open-metadata-implementation/view-services/glossary-author-view/glossary-author-view-spring/src/main/java/org/odpi/openmetadata/viewservices/glossaryauthor/server/GlossaryAuthorViewGlossaryRESTResource.java

```diff
@@ -164,6 +166,33 @@ public GlossaryAuthorViewGlossaryRESTResource() {
         return restAPI.getGlossaryRelationships(serverName, userId, guid, asOfTime, offset, pageSize, sequencingOrder, sequencingProperty);
     }
 
+    /**
+     * Create the supplied list of Terms in the glossary, identified by the supplied guid. Each term does not need to specify a glossary.
+     *
+     * @param serverName       local UI server name
+     * @param userId           user identifier
+     * @param guid             guid of the glossary under which the Terms will be created
+     * @param terms            terms to create
+     * @return a response which when successful contains a list of the responses from the Term creates (successful or otherwise). The order of the responses is the same as the supplied terms order.
+     *
+     * when not successful the following Exception responses can occur
+     * <ul>
+     * <li> UnrecognizedGUIDException            the supplied guid was not recognised</li>
+     * <li> UserNotAuthorizedException           the requesting user is not authorized to issue this request.</li>
+     * <li> FunctionNotSupportedException        Function not supported</li>
+     * <li> InvalidParameterException            one of the parameters is null or invalid.</li>
+     * <li> MetadataServerUncontactableException not able to communicate with a Metadata respository service.</li>
+     * </ul>
+     */
+    @PostMapping(path = "/{guid}/terms")
+    public SubjectAreaOMASAPIResponse<SubjectAreaOMASAPIResponse<Term>> createMultipleTermsInAGlossary(@PathVariable String serverName,
+                                                                                                   @PathVariable String userId,
+                                                                                                   @PathVariable String guid,
+                                                                                                   @RequestBody  Term[] terms
+                                                                                                  ) {
+        return restAPI.createMultipleTermsInAGlossary(serverName, userId, guid, terms);
```

Comment:
You need remove parameter "guid" after changes.

Reply:
As above - I think we need it.